packages/fxa-auth-server/scripts/prune-pocket-access-tokens.ts

```diff
@@ -0,0 +1,200 @@
+
+/*/
+  This script is used to perform a one-time cleanup of all Pocket OAuth
+  access tokens stored in fxa_oauth.tokens. Pocket was FxA's first OAuth client
+  and used a special workaround where access tokens were persisted in MySQL
+  instead of Redis to avoid implementing refresh tokens.
+
+  This script will:
+  1. Delete all access tokens for Pocket client IDs from the tokens table
+  2. Optionally run in dry-run mode to see what would be deleted
+/*/
+
+import { StatsD } from 'hot-shots';
+import program from 'commander';
+import { setupDatabase } from 'fxa-shared/db';
+import { BaseAuthModel } from 'fxa-shared/db/models/auth';
+
+const pckg = require('../package.json');
+const config = require('../config').default.getProperties();
+const statsd = new StatsD(config.statsd);
+const log = require('../lib/log')(
+  config.log.level,
+  'prune-pocket-access-tokens',
+  statsd
+);
+
+// Pocket client IDs from config
+const POCKET_CLIENT_IDS = [
+  '749818d3f2e7857f', // pocket-web
+  '7377719276ad44ee', // pocket-mobile
+];
+
+const DEFAULT_BATCH_SIZE = 1000;
+const DEFAULT_WAIT_MS = 1000;
+
+export async function init() {
+  program
+    .version(pckg.version)
+    .option(
+      '--dry-run [boolean]',
+      'Run in dry-run mode to see what would be deleted without actually deleting (default: true)',
+      true
+    )
+    .option(
+      '--batch-size <number>',
+      'Number of tokens to delete per batch (default: 1000)',
+      DEFAULT_BATCH_SIZE.toString()
+    )
+    .option(
+      '--wait <number>',
+      'Milliseconds to wait between batches in ms (default: 1000)',
+      DEFAULT_WAIT_MS.toString()
+    )
+    .on('--help', function () {
+      console.log(`
+Example:
+
+> ./scripts/prune-pocket-access-tokens.sh
+> ./scripts/prune-pocket-access-tokens.sh --dry-run=false --batch-size=500 --wait=2000
+
+Exit Codes:
+  0 - success
+  1 - unexpected error
+  2 - error during initialization`);
+    })
+    .parse(process.argv);
+
+  const dryRun = program.dryRun !== 'false' && program.dryRun !== false;
+  const batchSize = parseInt(program.batchSize) || DEFAULT_BATCH_SIZE;
+  const waitMs = parseInt(program.wait) || DEFAULT_WAIT_MS;
+  const sleep = () => new Promise((resolve) => setTimeout(resolve, waitMs));
+
+  log.info('Pocket token pruning start', {
+    dryRun,
+    batchSize,
+    waitMs,
+    pocketClientIds: POCKET_CLIENT_IDS,
+  });
+
+  // Setup Knex Connection
+  let db;
+  try {
+    db = setupDatabase({
+      ...config.oauthServer.mysql,
+    });
+
+    // Binds knex once, which effectively binds for all BaseAuthModel inherited types
+    BaseAuthModel.knex(db);
+  } catch (err) {
+    log.error('error during knex initialization', err);
+    return 2;
+  }
+
+  try {
+    for (const clientId of POCKET_CLIENT_IDS) {
+      const clientIdBuffer = Buffer.from(clientId, 'hex');
+
+      const countResult = await db('tokens')
+        .where('clientId', clientIdBuffer)
```

Comment:
There is a generic access token pruning [script](https://github.com/mozilla/fxa/blob/b1a42e5617c7a976f1fd35317f8e417004666654/packages/fxa-auth-server/scripts/prune-oauth-authorization-codes.ts#L11) that may also work? 

That script handles expiration, but in this case we are deleting all access tokens.